class Data_Exception(Exception):
    pass